images/plbase/plbase-install.sh

```diff
@@ -69,6 +72,7 @@ if [[ "${arch}" != "aarch64" ]]; then # R is not yet supported on ARM64.
         Rscript /r-requirements.R
     fi
```

Comment:
Can we remove the `if` statement here and unconditionally install `r-requirements.R`?

Reply:
Done in 06a2da4.